frontend/webpack.dev.js

```diff
@@ -0,0 +1,61 @@
+const path = require("path");
+const webpack = require("webpack");
+const { merge } = require("webpack-merge");
+
+const common = require("./webpack.config.js");
+
+// for testing: for prod, the Dockerfile should have the official prod version used
+const RWP_BASE_URL = process.env.RWP_BASE_URL || "https://replayweb.page/";
+const devBackendUrl = new URL(
+  process.env.API_BASE_URL || "https://btrix.webrecorder.net/api/"
```

Comment:
Should we not provide a default at all and throw an exception if its not set?

```suggestion
  process.env.API_BASE_URL || "https://btrix.webrecorder.net/api/"
  if (!process.env.API_BASE_URL) {
     throw new Error("Please set the API_BASE_URL in your .env.local to run a local server")
  }
  const devBackendUrl = new URL(process.env.API_BASE_URL);
```